pkg/component/component.go

```diff
@@ -37,6 +37,16 @@ type ComponentInfo struct {
 	Type string
 }
 
+// ComponentDesc holds all information about component
+type ComponentDesc struct {
```

Comment:
Is this the proper naming (IntelliJ complains that the name of struct starts with the name of the package)? Furthermore, is this the proper location for the struct?

Reply:
I think the location is OK, where else would you put it? Regarding the name, we have lots of similar "issues"… starting with the struct just above (`ComponentInfo`) so I'm not sure we should be too hung up on this, even though `component.Desc` reads better than `component.ComponentDesc`…

Reply:
I thought we had a package for models or something like that, but I admit that I haven't looked into it much

Reply:
done